modules/op-signing-secrets-in-tekton-chains.adoc

```diff
@@ -15,4 +15,19 @@ Currently, {tekton-chains} supports the `x509` and `cosign` signature schemes.
 Use only one of the supported signature schemes.
 ====
 
-To use the `x509` signing scheme with {tekton-chains}, store the `x509.pem` private key of the `ed25519` or `ecdsa` type in the `signing-secrets` Kubernetes secret.
+To use the `x509` signing scheme with {tekton-chains}, you must fulfill the following requirements:
+
+* Store the private key in the `signing-secrets` with the `x509.pem` structure.
+* Store the private key as an unencrypted `PKCS #8` PEM file: `BEGIN PRIVATE KEY`.
```

Comment:
What is `BEGIN PRIVATE KEY` here?

Reply:
It should refer to first lines of the `PKCS #8` PEM file. But I suppose it could cause more confusion that actually helping. So I think just mentioning `PKCS #8` should be enough. Thank you for pointing this out!